templates/components/password-reset-expired.jsx

```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var IconText = require('./icontext.jsx');
+
+var PasswordResetExpired = React.createClass({
+  render: function() {
+    return (
+      <IconText
+        className="messageBox"
+        iconClass="messageIcon fa fa-exclamation-triangle"
+        header="Seems that the link in your email expired."
+        headerClass="messageHeader">
+          <p>
+            Request a new reset password link below
```

Comment:
Good fix @Pomax. Following your lead with a few less words:

"Request a new link below to set your password"